cmd/assetsvc/handler_test.go

```diff
@@ -134,7 +134,6 @@ func Test_newChartResponse(t *testing.T) {
 			assert.Equal(t, cResponse.ID, tt.chart.ID, "chart ID should be the same")
 			assert.Equal(t, cResponse.Relationships["latestChartVersion"].Data.(models.ChartVersion).Version, tt.chart.ChartVersions[0].Version, "latestChartVersion should match version at index 0")
 			assert.Equal(t, cResponse.Links.(selfLink).Self, pathPrefix+"/charts/"+tt.chart.ID, "self link should be the same")
-			assert.Equal(t, len(cResponse.Attributes.(models.Chart).ChartVersions), len(tt.chart.ChartVersions), "number of chart versions in the response should be the same")
 			// We don't send the raw icon down the wire.
 			assert.Nil(t, cResponse.Attributes.(models.Chart).RawIcon)
```

Comment:
Do you think we should update this to include verifying the chart versions is empty?

Reply:
it's still not clear to me if the chart versions really need to be empty or if it's just our use case in which it's convenient that it's not included. Let's rethink about this in #1461